client/blocks/reader-subscription-list-item/index.jsx

```diff
@@ -15,6 +15,8 @@ import { getStreamUrl } from 'reader/route';
 import EmailSettings from './email-settings';
 import { getSiteName, getSiteUrl } from 'reader/get-helpers';
 
+const stripUrl = url => url.replace( 'https://', '' ).replace( 'http://', '' ).replace( /\/$/, '' );
```

Comment:
Decided against using it.  In this case its just for display purposes, `prepareComparableUrl` is for creating keys to the `reader.follows` redux subtree.  I think using the same func for both would be a bad idea so i reverted back to `stripUrl`